docs/admin/kubeadm.md

```diff
@@ -139,6 +151,37 @@ Here's an example on how to use it:
 By default, when `kubeadm init` runs, a token is generated and revealed in the output.
 That's the token you should use here.
 
+## Automating kubeadm
+
+While most of the tutorials for kubeadm are linear and involve copying the token from `kubeadm init` to each node, you can parallelize this for easier automation.  The only requirement is that you know the IP that master will have after it is started.
```

Comment:
"Rather than copying the token you obtained from `kubeadm init` to each node, as in the basic `kubeadm` tutorials, you can parallelize the token distribution for easier automation. To implement this automation, you must know the IP address that the master will have after it is started."